sympy/functions/special/bessel.py

```diff
@@ -593,6 +593,14 @@ def fdiff(self, argindex=2):
             self * (self.order + 1)/self.argument
 
 
+def _jn_orthopoly(n, z):
```

Comment:
You are right. I was not satisfied with the name either. I'll change it.

On Sat, Sep 5, 2015 at 3:01 PM, Kalevi Suominen <notifications@github.com>
wrote:

> In sympy/functions/special/bessel.py
> <https://github.com/sympy/sympy/pull/9893#discussion_r38810578>:
>
> > @@ -593,6 +593,14 @@ def fdiff(self, argindex=2):
> >              self * (self.order + 1)/self.argument
> >
> >
> > +def _jn_orthopoly(n, z):
>
> This name appears misleading. Though the coefficients fn for the
> spherical Bessel functions are located in orthopolys they are not
> actually polynomials. Neither are they orthogonal in any reasonable way.
> Perhaps just plain _jn could be sufficient?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/sympy/pull/9893/files#r38810578>.
>